package.json

```diff
@@ -66,7 +67,7 @@
     "trailingComma": "es5"
   },
   "lint-staged": {
-    "*.ts": [
+    "*packages/**/.ts": [
```

Comment:
yeah, I like it, we have test-harness in `ts`, but not forced linting. We might want to do that in the future though

Reply:
My intention is to enable it — but right now there is a lot the linter would probably complain.